.github/workflows/sbom-generation.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       uses: aws-actions/amazon-ecr-login@v1
 
     - name: Run Trivy vulnerability scanner
-      uses: aquasecurity/trivy-action@0.17.0
+      uses: aquasecurity/trivy-action@0.19.0
```

Comment:
I know that we run 0.20 in other places. See what the latest release is

Reply:
The latest release is 0.20. I will update that now

Reply:
The latest release is 0.24.0. I will update these jobs